sorting/go/bubble-sort.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"math"
+	"os"
+	"strconv"
+	"strings"
+)
+
+//Swaps two values of a slice
+func Swap(sliceInt []int, indexValue int) {
+	//Treats edge case
+	if indexValue < len(sliceInt)-1 {
+		tempValue := sliceInt[indexValue]
+		sliceInt[indexValue] = sliceInt[indexValue+1]
+		sliceInt[indexValue+1] = tempValue
+	}
+}
+
+//Recursive implementation of Bubble sort
+func bubbleSort(sliceInt []int) {
+	for k := 0; k < len(sliceInt)-1; k++ {
+		if sliceInt[k] > sliceInt[k+1] {
+			Swap(sliceInt, k)
```

Comment:
Why make a different function for swap? You can simple do it it one line like this `a, b = b, a`
Also the `if` condition in the swap function is always true because `k < len(sliceInt)-1` in the for loop
So I think you should remove `Swap` function and replace `Swap(sliceInt, k)` with `sliceInt[k], sliceInt[k+1] = sliceInt[k+1], sliceInt[k]`